src/main/kotlin/no/nav/syfo/domain/Personident.kt

```diff
@@ -8,3 +10,5 @@ value class Personident(val value: String) {
         }
     }
 }
+
+fun Personident.asProducerRecordKey(): String = java.util.UUID.nameUUIDFromBytes(value.toByteArray()).toString()
```

Comment:
Trenger kanskje ikke å ligge i domenet? 😊 Virker som veldig spesifikk funksjonalitet for kafka producer, så kan eventuelt ligge som en extension function der